addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5271,6 +5268,23 @@ registry.layout_column = SnippetOptionWidget.extend({
     },
 });
 
+registry.vAlignment = SnippetOptionWidget.extend({
+    /**
+     * @override
+     */
+    _computeWidgetState: function (methodName, params) {
+        if (methodName === 'selectClass') {
+            // If there is no `align-items-` class on the row, then the `align-
+            // items-stretch` class is selected, because the behaviors are
+            // equivalent in both situations.
+            if (!params.possibleValues.some(alignClass => this.$target[0].classList.contains(alignClass))) {
```

Comment:
@sobo-odoo Actually you might not need to do this check because it would just be enough to check if the "super" returns something or not